From 5155102dcd7762558e51b15bcd9616707959620c Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Wed, 13 Jul 2022 20:57:04 +0300
Subject: [PATCH 044/233] [FIO extras] imx: mx7ulp: disable u-boot NONSEC/PSCI
 if OPTEE

We use the PSCI implementation from OP-TEE by default, so don't
select these features if OP-TEE enabled.

Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 arch/arm/mach-imx/mx7ulp/Kconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-imx/mx7ulp/Kconfig b/arch/arm/mach-imx/mx7ulp/Kconfig
index f045707519d..f8be3827168 100644
--- a/arch/arm/mach-imx/mx7ulp/Kconfig
+++ b/arch/arm/mach-imx/mx7ulp/Kconfig
@@ -9,9 +9,9 @@ config LDO_ENABLED_MODE
 	  Select this option to enable the PMC1 LDO.
 
 config MX7ULP
-	select ARCH_SUPPORT_PSCI
-	select CPU_V7_HAS_NONSEC
-	select CPU_V7_HAS_VIRT
+	select ARCH_SUPPORT_PSCI if !OPTEE
+	select CPU_V7_HAS_NONSEC if !OPTEE
+	select CPU_V7_HAS_VIRT if !OPTEE
 	select HAS_CAAM
 	bool
 
-- 
2.25.1

